feat: Support translation of image URLs

resizoltan opened the pull request:
### Description:

This feature adds support for displaying images on the wiki that are located in `$MD_FOLDER/$GFX_PATH` directory. This is achieved by modifying every image URL that match the regex to point to the correct web URL of that image, for every copied file.

### Example:
I tested with the repo https://github.com/resizoltan/wiki_update (which I will take down after the PR)
#### Config:
```yaml
jobs:
  update-wiki:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@master
      - name: Update Wiki
        uses: resizoltan/wiki-page-creator-action@feature/gfx #  docker://decathlon/wiki-page-creator-action:latest
        env:
          ACTION_MAIL: resizoltan@gmail.com
          ACTION_NAME: resizoltan
          GH_PAT: ${{ secrets.WIKI_UPDATE_TOKEN }}
          MD_FOLDER: doc
          OWNER: resizoltan
          REPO_NAME: wiki_update
          GFX_PATH: gfx
```
#### URL modification:
!\[cat](gfx/cat.jpg) -> ! \[cat\](https://raw.githubusercontent.com/resizoltan/wiki_update/master/doc/gfx/cat.jpg)

### Notes:

- I had to change the image in action.yml for my fork to be able to test the changes. I am not sure what the correct approach here would be, I guess updating the docker image on the docker hub.
- Does not work for private repositories due to lack of access rights.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Decathlon/wiki-page-creator-action?pullRequest=20) <br/>All committers have signed the CLA.

resizoltan commented:
Added support for HTML-formatted image links (which support resizing):
\<img src="gfx/cat.jpg" width=200\> -> \<img src="https://raw.githubusercontent.com/resizoltan/wiki_update/master/doc/gfx/cat.jpg" width=200\>

Also fixed a minor bug in the original regex.

resizoltan commented:
Closing this pull request, since Andrew-Chen-Wang's github-wiki-action does this much more powerfully.
